userguide/intro.rst

```diff
@@ -357,7 +357,7 @@ U2
 * The :guilabel:`Endpoint does not support regions` and
   :guilabel:`Use v2 signatures` fields have been added to
```

Comment:
Please verify field label text and capitalization.

Reply:
Rechecked: it is correct for the legacy UI